components/review-sections/HouseholdReviewSection.tsx

```diff
@@ -1,11 +1,23 @@
 import React from "react";
+import PropTypes from "../../helpers/PropTypes";
 import useReviewSectionRows from "../../helpers/useReviewSectionRows";
 import householdSteps from "../../steps/household";
 import Storage from "../../storage/Storage";
 import { ReviewSection } from "../ReviewSection";
 
-export const HouseholdReviewSection: React.FunctionComponent = () => {
-  const rows = useReviewSectionRows(Storage.ProcessContext, householdSteps);
+interface Props {
+  readOnly?: boolean;
+}
+
+export const HouseholdReviewSection: React.FunctionComponent<Props> = (
+  props
+) => {
+  const { readOnly } = props;
+  const rows = useReviewSectionRows(
+    Storage.ProcessContext,
+    householdSteps,
+    Boolean(readOnly)
```

Comment:
I think this is clearer. It makes it obvious what the default value is.

```suggestion
    readOnly || false
```